src/v2/Apps/Auction/Components/AuctionDetails/SaleDetailTimer.tsx

```diff
@@ -66,21 +66,21 @@ export const getTimerCopy = (time, hasStarted, lotsAreClosing) => {
   else {
     // 1 hour or less
     if (parsedDays < 1 && parsedHours < 1 && parsedHours < 1) {
```

Comment:
`parsedHours < 1 && parsedHours < 1` is redundant if you want to clean that up. I know that's not this PR.

Reply:
wow so it is!